README.md

```diff
@@ -36,7 +36,7 @@ Getting started
 ---------------
 
 See the [online documentation](https://docs.micropython.org/) for API
```

Comment:
```suggestion
See the [online documentation](https://docs.micropython.org/) for the API
```

If we are changing to "reference" it seems like we should add "the" here to make it sound correct.